src/getting-started/README.md

```
@@ -8,21 +8,57 @@ Let's start by creating a simple plugin to learn how to develop Denops plugins.
 
 ## Create a Plugin
 
-Create a directory named `denops-getting-started` in your home directory and a
-file named `main.ts` within it, under `denops/denops-getting-started/`:
+Create a directory named `denops-getting-started` in your home directory with the following structure:
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)